waku/nwaku.go

```diff
@@ -1,10 +1,7 @@
 package waku
 
 /*
-	#cgo LDFLAGS: -L../third_party/nwaku/build/ -lwaku
-	#cgo LDFLAGS: -L../third_party/nwaku -Wl,-rpath,../third_party/nwaku/build/
-
-	#include "../third_party/nwaku/library/libwaku.h"
+	#include "libwaku.h"
```

Comment:
I think is a bit more idiomatic in that way so that the reader know this thingy comes from compile flag `-I...`
```suggestion
	#include <libwaku.h>
```